Packs/Troubleshoot/IncidentFields/incidentfield-Instance_Name.json

```diff
@@ -0,0 +1,34 @@
+{
+ "associatedToAll": true,
+ "caseInsensitive": true,
+ "cliName": "instancename",
+ "closeForm": false,
+ "content": true,
+ "description": "Name of the instance to troubleshoot",
+ "editForm": true,
+ "fieldCalcScript": "096f78c3-e8a5-42ee-82e5-1540fc285d3f",
+ "group": 0,
+ "hidden": false,
+ "id": "incident_instancename",
+ "isReadOnly": false,
+ "locked": false,
+ "name": "Instance Name",
+ "neverSetAsRequired": false,
+ "ownerOnly": false,
+ "propagationLabels": [
+  "all"
+ ],
+ "required": false,
+ "selectValues": [
+  ""
```

Comment:
why is this empty?

Reply:
because it filled with a script

Reply:
but why is an empty string required here?

Reply:
🤷